datatypes/src/commonMain/kotlin/at/asitplus/crypto/datatypes/ECPoint.kt

```diff
@@ -87,7 +90,11 @@ sealed class ECPoint private constructor(
         else if (this is Normalized)
             "ECPoint[$curve]: (${homX.toString(16)} : ${homY.toString(16)}) [normalized]"
         else
-            "ECPoint[$curve]: (${(homX/homZ).toString(16)} : ${(homY/homZ).toString(16)}) [with Z = ${homZ.toString(16)}]"
+            "ECPoint[$curve]: (${(homX / homZ).toString(16)} : ${(homY / homZ).toString(16)}) [with Z = ${
+                homZ.toString(
+                    16
+                )
+            }]"
```

Comment:
this indentation change feels particularly excessive (and quite a few of the others make the code less readable imo, too)